charts/minikube.md

```diff
@@ -60,30 +63,66 @@ helm install --name zenko \
 
 To view the Kubernetes dashboard type the following and will launch the dashboard in your default browser:
 ```shell
-minikube dashboard
+$ minikube dashboard
 ```
 #### NOTE that once you helm install, it may take several minutes for all the pods to load and stabilize
 
 The endpoint can now be accessed via the kubernetes cluster ip (run ```minikube ip``` to display the cluster ip)
 
+## Testing your deployment
+
+To test your minikube deployment, please refer to our client configuration guide,
+from our CloudServer documentation. By default, minikube only exposes SSL port
+443, so you'll want to ask your client/app to use SSL. However, since minikube
+uses a self-signed certificate, you may get security error. You can either
+configure minikube to use a trusted certificate, or simply ignore the
+certificate.
+
+See the example below
+
+```shell
+$ cat /etc/hosts
+127.0.0.1   localhost
+127.0.1.1   machine
+
+192.168.99.100 minikube
```

Comment:
Just a heads up that in this configuration the hosts file modification isn't necessary and should be able to connect directly using the IP address because of the way we setup the hosts in `single-node-values.yml`. Not a blocker but just incase we wanted to add that.

Reply:
I can add a mention ; the main thing, to me, is that it's easier then for Orbit config because endpoint definition DNS style is human-readable and less error prone than IPs. I'll add a note explaining why this choice is made in the tutorial, good catch!